### PR TITLE
Add MercureBundle recipe

### PR DESCRIPTION
**Pull request by dunglas**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: symfony/mercure-bundle/0.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\MercureBundle\\MercureBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "container": {
+        "env(MERCURE_PUBLISH_URL)": "",
```
Comment (Pierstoval) [inline symfony/mercure-bundle/0.1/manifest.json]: Why add them as defaults in the container when you have them set up in env? 🤔

Comment (dunglas) [reply symfony/mercure-bundle/0.1/manifest.json]: To prevent an error when env var are not defined (when building a Docker container for instance).

Comment (sroze) [reply symfony/mercure-bundle/0.1/manifest.json]: You can instead define the `env(MERCURE_PUBLISH_URL)` parameter in the bundle.

Comment (dunglas) [reply symfony/mercure-bundle/0.1/manifest.json]: It’s not the bundle’s responsibility: you may use it without env cars.
Also, the use may choose to use other names of env vars.

**Pull request merged**